tools/ci_build/github/azure-pipelines/linux-openvino-ci-pipeline.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         pythonInterpreter: '/usr/bin/python3'
         workingDirectory: $(Build.BinariesDirectory)
 
-    - script: 'tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d openvino -r $(Build.BinariesDirectory) -x "--use_openvino CPU_FP32 --build_wheel"'
+    - script: 'tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d openvino -r $(Build.BinariesDirectory) -x "--use_openvino CPU_FP32 --build_wheel --use_full_protobuf"'
```

Comment:
I think you were asking about ubuntu 18.04 at one point. 
are we okay with keeping testing primarily on ubuntu 16.04 ?

Reply:
Ubuntu 16.04 should still be good for now for minimum required version as the all our dependency packages are still supported on it.
 Moving to 18.04 may become a minimum requirement sometime in the future if any of our dependencies require it. We will keep a look out for changes going forward.